app-modules/pipeline/src/Filament/Pages/PipelineSettings.php

```diff
@@ -64,7 +67,7 @@ public static function canAccess(): bool
         /** @var User $user */
         $user = auth()->user();
 
-        if (! $user->hasLicense(Prospect::getLicenseType())) {
+        if (! $user->hasLicense(Student::getLicenseType()) || ! $user->hasLicense(Prospect::getLicenseType())) {
```

Comment:
Sorry, reading this now I am realizing actually this should probably be AND (`&&`) since w are trying to say. They don't have access if they don't have either license.